gradle/verification-metadata.xml

```diff
@@ -7,8 +7,7 @@
          <trust file=".*-javadoc[.]jar" regex="true"/>
          <trust file=".*-sources[.]jar" regex="true"/>
          <trust file=".*-src[.]zip" regex="true"/>
-         <trust group="net[.]rubygrapefruit" name="native-platform.*" regex="true"/>
-         <trust group="net[.]rubygrapefruit" name="file-events.*" regex="true"/>
+         <trust group="net.rubygrapefruit"/>
```

Comment:
Why do we not need the `[]` around the dot anymore?

Reply:
Because the `group` name isn't treated as a regex any more, since the `regex="true"` tag is gone.